packages/web-components/src/components/ic-back-to-top/ic-back-to-top.tsx

```diff
@@ -42,7 +42,13 @@ export class BackToTop {
   };
 
   private setFooterVisible = (visible: boolean) => {
-    this.footerVisible = visible;
+    this.checkForClassificationBanner();
+    if (typeof window !== "undefined" && window.scrollY === 0) {
```

Comment:
Even though it's against our guidance, what happens if the footer is not below the fold but window.scrollY is 0?

Reply:
The back to top would not be displayed at all, as the target el would not have scrolled off screen yet